tests/test_opencode_plugin_install.py

```
@@ -155,6 +155,8 @@ def main() -> int:
         check_env = env.copy()
         check_env["CMUX_TEST_OPENCODE_PLUGIN_PATH"] = str(plugin_path)
         check_env["CMUX_TEST_OPENCODE_PLUGIN_COPY_PATH"] = str(plugin_copy_path)
+        check_env["CMUX_TEST_OPENCODE_FEED_PLUGIN_PATH"] = str(feed_plugin_path)
+        check_env["CMUX_SOCKET_PATH"] = f"/tmp/cmux-opencode-feed-{os.getpid()}.sock"
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-opencode-feed-"

(S108)